README.md

```diff
@@ -84,8 +99,25 @@ uses: buildpacks/github-actions/setup-pack
 | `pack-version` | Optional version of [`pack`][pack] to install. Defaults to latest release.
 | `yj-version` | Optional version of [`yj`][yj] to install. Defaults to latest release. 
 
+## Verify Buildpackage Action
+The verify-buildpackage action parses the metadata on a buildpackage and verifies that the `id` and `version` match expected values.
+
+```yaml
+uses: docker://ghcr.io/buildpacks/actions/verify-buildpackage
+with:
+  id:      test-buildpack
+  version: "1.0.0"
+  address: ghcr.io/buildpacks/test-buildpack@sha256:04ba2d17480910bd340f0305d846b007148dafd64bc6fc2626870c174b7c7de7
```

Comment:
```suggestion
  address: ghcr.io/example/test-buildpack@sha256:04ba2d17480910bd340f0305d846b007148dafd64bc6fc2626870c174b7c7de7
```

not sure this matter much. we could also publish a test-buildpack so it actually works